go.mod

```diff
@@ -50,6 +49,7 @@ require (
 	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
 	github.com/modern-go/reflect2 v1.0.2 // indirect
 	github.com/munnerz/goautoneg v0.0.0-20191010083416-a7dc8b61c822 // indirect
+	github.com/pkg/errors v0.9.1 // indirect
```

Comment:
Do you know which package this is coming from? We can raise an issue to let the owners know.

Reply:
Will find out separately 👍🏻